.expeditor/verify.pipeline.yml

```diff
@@ -190,4 +190,23 @@ steps:
         docker:
           privileged: true
           environment:
-            - HAB_ORIGIN=ci
\ No newline at end of file
+            - HAB_ORIGIN=ci
+
+  - label: "[build] :linux: builder-api-proxy"
+    command:
+      - hab pkg build components/builder-api-proxy/habitat/
+    retry:
+      automatic:
+        limit: 1
+    expeditor:
+      executor:
+        docker:
+          privileged: true
+          environment:
+            - HAB_ORIGIN=habitat
+            - HAB_INTERNAL_BLDR_CHANNEL=LTS-2024
+            - HAB_BLDR_CHANNEL=LTS-2024
+            # TODO : This needs to be moved to LTS-2024 as well in future. Right now
+            # This is done because some of our `hab` packages like `hab-studio` are
+            # not available on `LTS-2024` yet.
+            - HAB_FALLBACK_CHANNEL=stable
```

Comment:
I don't think the fallback channel is going to help here. `stable` is already the default but this variable is only used for a hab build and not installing missing hab components before entering the studio which is the case here.

I think (hope) the solution here is to leave `HAB_BLDR_CHANNEL` alone which will get the studio from stable. Then when it enters the studio and starts the build it will use `HAB_INTERNAL_BLDR_CHANNEL` to set `HAB_BLDR_CHANNEL` inside of the studio.

Reply:
@mwrock : Wondering if it's a better idea to pass `HAB_STUDIO_SECRET_HAB_BLDR_CHANNEL` and `HAB_STUDIO_SECRET_HAB_FALLBACK_CHANNEL` (both set to LTS-2024) `env` variables? This makes sure inside the `studio` accidentally `stable` does not get used at all?

Reply:
I am able to make progress in the build with the two env variables above, still seeing some issues, but don't look related to `hab` or `studio`

Reply:
Oh man I'm sorry. You are absolutely correct. `HAB_STUDIO_SECRET_HAB_BLDR_CHANNEL` was the correct variable. I'm not sure setting `HAB_STUDIO_SECRET_HAB_FALLBACK_CHANNEL` to stable will do anything because I think that is the default anyways.

Reply:
Nope: I meant setting `HAB_STUDIO_SECRET_HAB_FALLBACK_CHANNEL` to `LTS-2024` so that, we know inside studio `stable` is not accidentally used somehow.